src/SFA.DAS.Commitments.Database/StoredProcedures/UpdateDatalockStatusIsExpired.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [dbo].[UpdateDatalockStatusIsExpired]
+	@ApprenticeshipId bigint ,
+	@PriceEpisodeIdentifier NVARCHAR(255)
+	WITH RECOMPILE
```

Comment:
Is there a reason for this option?

Reply:
Advice here: https://www.mssqltips.com/sqlservertip/3257/different-approaches-to-correct-sql-server-parameter-sniffing/

Basically its a judgement call as to if this would be needed. My choice to use it is influenced by enforced corporate standards , that have become habitual.  Would be needed if poor performance reared its head later due to operational conditions. Happy for the RECOMPILE to come out., and leave such issues to DevOps if performance related tickets are raised.